<!--StartFragment-->

<h3 align="center">
<kbd>TRACK: AMR IN CANCER</kbd>
<br/>
LITERATURE REVIEW OF
<br/>
Whole-Genome Sequencing Accurately Identifies Resistance to Extended </br> 
Spectrum β-Lactams for Major Gram-Negative Bacterial Pathogens
</h3>

<h4 align="center">
AUTHORS
</h4> 
<div align="center">
<table><tr><td> 
  
Adaradohun Samson (***@Adara***); Nwokocha Amarachi (***@Amara***); </br>
Lakshana Bakthavachalam (***@Lakshana***); Nada ElSayed Ahmed (***@Nada\_EA***)

</td></tr></table>
</div>

***

<br/>
<h3 align="center">
BACKGROUND
</h3> 
The increasing prevalence of antimicrobial resistance (AMR) presents a major challenge in managing bacterial infections. Conventional methods for AMR detection, such as phenotypic assays, are often time-consuming and limited in their capacity to identify all resistance mechanisms. Whole-genome sequencing (WGS) emerges as a promising alternative, offering comprehensive insights into both established and novel resistance factors. This study focuses on neutropenic cancer patients, who are particularly vulnerable to infections. The core objective of the study was to assess WGS’s effectiveness in predicting resistance among major gram-negative pathogens to β-lactam antibiotics commonly used in empiric treatment for neutropenic fever.

<br/>
<h3 align="center">
STUDY DESIGN
</h3> 
The samples consisted of 90 bacterial isolates collected from blood samples of cancer patients suffering from febrile neutropenia. To detect AMR, Illumina MiSeq WGS was employed to identify gene presence/absence and mutations in AMR-relevant genes. Antimicrobial susceptibility testing was carried out by determining minimum inhibitory concentrations (MICs) for four β-lactam antibiotics (ceftazidime, cefepime, piperacillin-tazobactam, and meropenem) using broth microdilution (BMD), the gold standard for AMR detection. Additionally, commercial methods such as Etest and Vitek 2 were used for comparison. WGS was then compared to both BMD and commercial methods to evaluate accuracy and diagnostic performance.

<br/>
<h3 align="center">
INSIGHTS AND FINDINGS
</h3>
The main findings showed that antimicrobial resistance in important gram-negative bacteria, such as _Escherichia coli_, _Pseudomonas aeruginosa_, _Klebsiella pneumoniae_, and _Enterobacter cloacae_, was highly predictable using WGS. The study reported that WGS identified AMR with sensitivity, specificity, and positive predictive values of 0.87, 0.98, and 0.97, respectively. These gram-negative bacteria exhibited a range of resistance mechanisms to antibiotics like ceftazidime, cefepime, and piperacillin-tazobactam. While WGS data demonstrated a reduction in positive predictive value and specificity compared to MICs from commercial methods, WGS still detected 35% more resistant isolates than PCR. This highlights WGS as a valuable technique for identifying AMR in clinical settings, especially in high-risk patients like those with neutropenia.

<br/>
<h3 align="center">
BOUNDARIES OF RESEARCH
</h3>
Limitations of the study that warrant consideration includes, data from genotypic information that might not reflect transcript-level changes, time-consuming manual sequence inspection for SNPs and indels identification and limiting the genotypic prediction to 4 broad-spectrum beta-lactams, excluding a broader array of antimicrobials.

<br/>
<h3 align="center">
CONCLUDING THOUGHTS AND TAKEAWAYS
</h3>
To summarize, the evolution of WGS improves the feasibility of genome-wide analysis in clinical microbiology, as DNA extraction, sequencing, and analysis are less resource-intensive compared to current techniques. Nonetheless, thorough characterization and testing of genetic basis of resistance for all clinically relevant antimicrobials are crucial to achieving this goal.

<br/>
<h3 align="center">
REFERENCE
</h3>
Shelburne, S. A., Kim, J., Munita, J. M., Sahasrabhojane, P., Shields, R. K., Press, E. G., Li, X., Arias, C. A., Cantarel, B., Jiang, Y., Kim, M. S., Aitken, S. L., & Greenberg, D. E. (2017). Whole-Genome Sequencing Accurately Identifies Resistance to Extended-Spectrum β-Lactams for Major Gram-Negative Bacterial Pathogens. _Clinical infectious diseases : an official publication of the Infectious Diseases Society of America_, _65_(5), 738–745. (https://doi.org/10.1093/cid/cix417)

<br/><br/>
***VIDEO LINK***: *(https://www.linkedin.com/posts/nada-elsayed-ahmed_hackbio2024-amrincancer-bioinformatics-activity-7238205554685284352-yYro?utm_source=share&utm_medium=member_desktop)*

<!--EndFragment-->
