    1  ~
    2  q
    3  pwd
    4  clear
    5  pwd
    6  ls
    7  ls -l
    8  ls -lh
    9  ls -a
   10  ls -a -l
   11  -lS
   12  -ls -lS
   13  -ls -a -R
   14  pwd
   15  clear
   16  -ls
   17  ls
   18  ls -lS
   19  ls -a -lS
   20  ls -a -R
   21  ls -a
   22  ls -a -lh
   23  pwd
   24  mkdir IBT_2024
   25  ls
   26  ls -a
   27  ls -a -R
   28  rmdir IBT_2024
   29  ls
   30  mkdir ibt2024
   31  ls
   32  cd ./ibt2024
   33  pwd
   34  mkdir ../assignments
   35  ls ..
   36  ls -l ~/home
   37  ls -l /home
   38  ls -l /home/nada_ahmed
   39  ls -lh /home/nada_ahmed
   40  ls
   41  cd ..
   42  ls
   43  pwd
   44  cd ibt2024/
   45  pwd
   46  mkdir module2
   47  ls
   48  cd module2/
   49  pwd
   50  mkdir session1 session2
   51  ls
   52  cd ..
   53  ls
   54  pwd
   55  cd ~
   56  pwd
   57  ls
   58  ls ./ibt2024
   59  ls ./ibt2024/module2/
   60  rm ibt2024
   61  rm assignments/
   62  rmdir ibt2024/
   63  rmdir assignments/
   64  ls
   65  rmdir -r ibt2024/
   66  rmdir -R ibt2024/
   67  ls -R ibt2024/
   68  man rmdir
   69  man rmdir -r
   70  pwd
   71  ls
   72  ls -a
   73  ls -a -r
   74  ls -a -R
   75  cd ./ibt2024
   76  ls
   77  ls -a
   78  man ls
   79  man pwd
   80  pwd
   81  cd
   82  pwd
   83  cd ibt2024/
   84  cd ~nada_ahmed
   85  pwd
   86  cd ./ibt2024/module2/session1
   87  cd ../session2
   88  cd ..
   89  cd session
   90  cd ./sessiona
   91  cd ./session1
   92  cd ../session2
   93  clear
   94  pwd
   95  man pwd
   96  pwd -version
   97  pwd --version
   98  pwd -L
   99  pwd -P
  100  man pwd
  101  pwd --logical
  102  /bin/pwd --version
  103  pwd --help
  104  pwd --version
  105  man pwd
  106  rmdir -R ibt2024
  107  man rmdir
  108  rmdir -r ibt2024
  109  rmdir ibt2024
  110  ls
  111  pwd
  112  mkdir Coursera MasterCourses IBT2016
  113  ls
  114  mkdir ./IBT2016/Genomics ./IBT2016/Linux
  115  cd ./IBT2016/
  116  ls
  117  mkdir ./Genomics/VCFfiles ./Genomics/Fastafiles ./Linux/'Session 1' ./Linux/'Session 2'
  118  ls ./Genomics/
  119  ls ./Linux/
  120  mkdir ./Linux/Session 1
  121  ls ./Linux/
  122  mkdir ./Linux/Session\ 1
  123  mkdir ./Linux/Session\ 3
  124  ls ./Linux/
  125  cd ./Linux/
  126  ls
  127  mkdir 'Session 1'
  128  mkdir 'Session 2'
  129  ls
  130  ls -R
  131  cd ~
  132  pwd
  133  mkdir Coursera MasterCourses IBT2016
  134  ls
  135  cd IBT2016/
  136  mkdir Genomics Linux
  137  ls
  138  cd Genomics/
  139  mkdir VCFfiles Fastafiles
  140  ls
  141  cd ../Linux/
  142  mkdir 'Session 1' 'Session 2'
  143  ls
  144  cd ../../Coursera/
  145  mkdir FirstCourse
  146  ls
  147  cd firstcourse
  148  rmdir FirstCourse
  149  ls
  150  cd ../IBT2016/Genomics/
  151  ls -l
  152  rmdir Fastafiles/
  153  ls
  154  mkdir ../Linux/'Session 3'
  155  ls ../Linux/
  156  cd ~
  157  pwd
  158  cd ./IBT2016/Linux/
  159  ls
  160  cd Session\ 1
  161  nano test_nano
  162  ls
  163  nano test_nano 
  164  ls
  165  nano file2
  166  ls
  167  nano file2 
  168  nano test_nano
  169  nano test_nano 
  170  ls
  171  rm test_nano 
  172  ls
  173  nano file2
  174  ls -l
  175  ls -a
  176  nano file2
  177  cat file2
  178  q
  179  nano file2
  180  cat file2
  181  more file2
  182  less file2
  183  head file2
  184  tail file2
  185  less file2
  186  man mutt
  187  man mut
  188  head -2 file2
  189  tail -3 file2
  190  head -4 file2
  191  cp -i file1 file3
  192  ls
  193  nano file3
  194  cp -i file3 file2
  195  ls
  196  nano file2
  197  clear
  198  cp fil3 ../Session\ 3
  199  cp file3 ../Session\ 3
  200  ls ../Session\ 3
  201  cp . ../Session\ 2
  202  cp ../Session\ 1 ../Session\ 2
  203  cp -R ../Session\ 1 ../Session\ 2
  204  ls ../Session\ 2
  205  mkdir ~/test
  206  ls ~
  207  mv -i ~/test/ ~/MasterCourses/
  208  ls ~
  209  ls ~/MasterCourses/
  210  mkdir test2
  211  mv test2 ~/MasterCourses/
  212  ls ~/MasterCourses/
  213  mv ~/MasterCourses/test ~/MasterCourses/test2
  214  ls ~/MasterCourses/
  215  mkdir test3
  216  mv -i test3 ~/MasterCourses/test2/
  217  ls
  218  ls ~/MasterCourses/
  219  ls ~/MasterCourses/test2/
  220  ls ~/IBT2016/Linux/
  221  ls ~/IBT2016/Linux/Session\ 1
  222  mv -i file1 file2
  223  ls
  224  cat file2 
  225  ls
  226  rm -i file
  227  rm -i file2 
  228  ls
  229  ls ~/MasterCourses/
  230  rm test2
  231  rm ~/MasterCourses/test2/
  232  rm -r ~/MasterCourses/test2/
  233  ls ~/MasterCourses/
  234  nano ../Session\ 2
  235  nano ../Session\ 2/file1
  236  ls
  237  nano file2
  238  nano test_file
  239  nano file4
  240  ls
  241  mv test_file file3
  242  ls
  243  nano file3
  244  mv -i file4 File4
  245  ls
  246  [[::upper::]]*
  247  mkdir session3 ~/IBT2016/Linux/
  248  ls ~/IBT2016/Linux/
  249  cp [[::upper::]]
  250  cat [:upper:]
  251  cd ../../Genomics/
  252  wget hmp://plasmodb.org/common/downloads/
  253  release-9.0/Pfalciparum/fasta/
  254  PlasmoDB-9.0_Pfalciparum_BarcodeIsolates.fasta
  255  wget hmp://plasmodb.org/common/downloads/release-9.0/Pfalciparum/fasta/PlasmoDB-9.0_Pfalciparum_BarcodeIsolates.fasta
  256  wget http://plasmodb.org/common/downloads/release-9.0/Pfalciparum/fasta/PlasmoDB-9.0_Pfalciparum_BarcodeIsolates.fasta
  257  ls
  258  cat PlasmoDB-9.0_Pfalciparum_BarcodeIsolates.fasta [B
  259  tail -5 PlasmoDB-9.0_Pfalciparum_BarcodeIsolates.fasta 
  260  less PlasmoDB-9.0_Pfalciparum_BarcodeIsolates.fasta 
  261  less PlasmoDB-9.0_Pfalciparum_BarcodeIsolates.fasta
  262  more PlasmoDB-9.0_Pfalciparum_BarcodeIsolates.fasta
  263  less PlasmoDB-9.0_Pfalciparum_BarcodeIsolates.fasta
  264  mv ../Linux/Session\ 1 ../Linux/Session1
  265  ls ../Linux/
  266  mv ../Linux/Session\2 ../Linux/Session2
  267  mv ../Linux/Session\ 2 ../Linux/Session2
  268  ls ../Linux/
  269  ls -R ../Linux/Session1
  270  ls -R ../..
  271  mkdir Fastafiles
  272  ls
  273  cd
  274  ls
  275  ls ./IBT2016/
  276  ls ./IBT2016/Linux/
  277  mkdir ~/IBT2016/Linux/Session2/assignment
  278  ls ./IBT2016/Linux/Session2
  279  cd ~/IBT2016/Linux/Session2/assignment
  280  nano genes1 genes2
  281  cat genes1 genes2
  282  cp -i genes1 ../../Session1
  283  ls ../../Session1
  284  cat ../../Session1/genes1 
  285  mv ../../Session1/genes1 ../../Session1/test1
  286  ls ../../Session1/
  287  tail -n2 genes2
  288  head genes1
  289  touch genes3 genes4 genes5
  290  ls
  291  cp g* ../../Session1
  292  ls ../../Session1
  293  ls -R
  294  cd
  295  ls -R
  296  cd ~/IBT2016/Linux/Session2/
  297  rm -r assignment/
  298  ls
  299  ls -R ~
  300  ls -R
  301  cat ./IBT2016/Linux/Session1/test1 
  302  wc ./IBT2016/Linux/Session1/test1 
  303  wc -m ./IBT2016/Linux/Session1/test1 
  304  wc -c ./IBT2016/Linux/Session1/test1 
  305  wc -l ./IBT2016/Linux/Session1/test1 
  306  man wc
  307  wc -w ./IBT2016/Linux/Session1/test1 
  308  cd ./IBT2016/Linux/Session1
  309  ls
  310  cat genes1
  311  cat genes2
  312  cat genes3 genes4 genes5 test1 
  313  cat test1 
  314  sort test1 
  315  join genes1 genes2
  316  diff genes1 genes2
  317  nano test1 
  318  cat test1 
  319  sort test1 
  320  uniq test1 
  321  man uniq
  322  nano test1 
  323  cat test1 
  324  uniq test1 
  325  man sort
  326  cat test1 
  327  sort test1 
  328  nano test1 
  329  cat test1 
  330  sort test1 
  331  cat test1 
  332  sort -k 2 test1 
  333  sort -t $'\t' test1 
  334  man sort
  335  grep 1 test1 
  336  grep -v 1 test1 
  337  nano test1 
  338  grep -v 1 test1 
  339  cat test1 
  340  cut -d' ' -f2 test1 
  341  cut -d' ' -f1 test1 
  342  cut -d' ' -f2,1 test1 
  343  uniq test1 
  344  man uniq
  345  uniq [sort test1 ]
  346  uniq [sort test1]
  347  ls
  348  ls -l
  349  ls -lh
  350  cat test1 
  351  sort test1 > sorted_test1
  352  ls
  353  cat sorted_test1 
  354  uniq test1 
  355  uniq sorted_test1 
  356  cat test1 
  357  sort test1 
  358  uniq -c sorted_test1 
  359  cat genes1
  360  join genes1 sorted_test1 
  361  cat genes1
  362  cat sorted_test1 
  363  cat genes1
  364  cat sorted_test1 
  365  join sorted_test1 genes1
  366  cat genes2
  367  join genes1 genes2
  368  diff genes1 genes2
  369  cat genes1
  370  cat genes2
  371  diff genes1 genes2
  372  diff -help
  373  diff --help
  374  man diff
  375  diff genes1 genes2
  376  cat genes1 genes2
  377  ls -R ~
  378  mkdir ../Session3
  379  cd ../Session3
  380  nano fileA fileB
  381  cd ../Session1
  382  ls
  383  cat genes1 genes2
  384  cat test1 
  385  sort test1
  386  uniq sorted_test1 
  387  uniq -c sorted_test1 
  388  uniq -c sorted_test1 > uniq_test1
  389  cat uniq_test1 
  390  cd ../Session3
  391  ls
  392  join fileA fileB > myfirstjoin
  393  sort fileA > SortedA
  394  sort fileB > SortedB
  395  nano fileA fileB
  396  mv fileA FileA
  397  mv fileB FileB
  398  ls
  399  cat myfirstjoin 
  400  cat FileA
  401  cat FileB
  402  ls
  403  cat SortedA
  404  cat SortedB
  405  join SortedA SortedB > JoinAB
  406  cat JoinAB 
  407  uniq SortedA
  408  uniq -c SortedA
  409  uniq -c SortedA > SortedA_count
  410  cat SortedA_count 
  411  ls
  412  nano testfasta
  413  cat testfasta 
  414  nano testfasta
  415  wc -l testfasta 
  416  grep '>' testfasta 
  417  ls
  418  uniq -c SortedA
  419  uniq -c SortedA | wc -l
  420  grep -c '>' testfasta 
  421  cat testfasta 
  422  grep -c '>' testfasta 
  423  grep -o '>' testfasta | wc -l
  424  grep --help
  425  ls -l
  426  ls -ltr
  427  ls -lt
  428  cat testfasta 
  429  nano testfasta 
  430  wc -l testfasta 
  431  grep -c '>' testfasta 
  432  nano testfasta 
  433  printenv
  434  cd ../Session2
  435  printenv
  436  ls
  437  cd ..
  438  ls
  439  ls -l
  440  HOME
  441  printenv
  442  cd ./Session1
  443  ls -l
  444  chmod u=xrw,g=rw,o=r genes1
  445  ls -l
  446  chmod 754 genes3
  447  ls -l
  448  printenv HOME
  449  printenv PWD
  450  printenv PATH
  451  history
  452  echo test > test.txt
  453  ls
  454  cat test.txt 
  455  echo Bioinrofmatics is great > bioinformatics
  456  ls
  457  cat bioinformatics 
  458  rm bioinformatics test.txt 
  459  ls
  460  echo Bioinfromatics is great \n Start > file
  461  cat file
  462  echo -e Bioinformatics is great \n Start > file2
  463  cat file2
  464  echo -e 'Bioinformatics is great \n Start' > file3
  465  cat file3
  466  rm file file2 file3
  467  ls
  468  temperature=15
  469  echo temperature
  470  $echo temperature
  471  echo $temperature 
  472  ls
  473  degree=18/18
  474  echo $degree 
  475  echo '$degree'
  476  echo "$degree"
  477  echo `pwd`
  478  echo $(pwd)
  479  echo HOME
  480  echo $HOME
  481  echo $pwd
  482  echo `pwd`
  483  echo $PWD
  484  echo $PATH
  485  cd ..
  486  echo $PWD
  487  pwd
  488  $echo $PWD
  489  nano myfirstscript
  490  ./myfirstscript
  491  ls -l
  492  chmod 744 myfirstscript 
  493  ./myfirstscript 
  494  cat myfirstscript 
  495  nano myfirstscript 
  496  ./myfirstscript 
  497  pwd
  498  ls
  499  ls -l
  500  nano myfirstscript 
  501  ./myfirstscript 
  502  vat myfirstscript 
  503  cat myfirstscript 
  504  echo $PATH
  505  chmod g+w myfirstscript 
  506  ls -l
  507  chmod g-w o+w myfirstscript 
  508  chmod o
  509  o+w myfirstscript 
  510  chmod o+w myfirstscript 
  511  ls -l
  512  chmod o-w | g+w myfirstscript 
  513  ls -l 
  514  chmod 744 myfirstscript 
  515  ls -l
  516  which perl
  517  which bash
  518  nano myfirstscript 
  519  ./myfirstscript 
  520  nano myfirstscript 
  521  ./myfirstscript 
  522  cat myfirstscript 
  523  which bash
  524  ls -l 
  525  nano file
  526  ls -l
  527  rm file 
  528  ls -l
  529  nano trial
  530  which bash
  531  nano trial 
  532  ls
  533  ls -l
  534  chmod u+x trial 
  535  ls -l
  536  ./trial 
  537  cat trial 
  538  nano trial 
  539  ./trial 
  540  nano trial 
  541  ./trial 
  542  nano trial 
  543  ls
  544  ls -l
  545  find *.txt
  546  nano file1.txt
  547  nano file2.txt
  548  find *.txt
  549  find *.txt | wc-l
  550  find *.txt | wc -l
  551  nano trial 
  552  ./trial 
  553  nano trial 
  554  ./trial 
  555  cat tr
  556  fastqc --help
  557  sudo apt install fastqc
  558  fastqc --help
  559  sudo apt install fastqc
  560  pwd
  561  ls
  562  ls -l
  563  cat test1 test2
  564  rm test1 test2
  565  ls
  566  ls ./test/
  567  ls ./test/data
  568  ls ./test/Data
  569  rmdir ./test/Data/
  570  rmdir test/
  571  ls
  572  cd IBT2016/
  573  ls
  574  cd 
  575  ls
  576  mkdir module5
  577  ls
  578  cd module5/
  579  pwd
  580  wget https://vula.uct.ac.za/access/content/attachment/f49e631d-97a2-4e5f-9738-1ba53bf4c9fb/Assignments/d80d4a05-6ecf-4168-b860-bbff9447a33e/test_Session1%20_2_.vcf
  581  ls -l
  582  mv test_Session1\ _2_.vcf test_Session1\_2_.vcf 
  583  ls
  584  head test_Session1_2_.vcf 
  585  head -20 test_Session1_2_.vcf 
  586  tail test_Session1_2_.vcf 
  587  more test_Session1_2_.vcf 
  588  head --help
  589  ls
  590  rm test_Session1_2_.vcf 
  591  ls
  592  ls -l
  593  alias rzi="rm -rf **/*Zone.Identifier
  594  test_Session1.vcf:z
  595  pwd
  596  cd
  597  exit
  598  ls
  599  head test_Session1.vcf
  600  head -20 test_Session1.vcf
  601  grep CTAGAG test_Session1.vcf
  602  grep CTAGAG test_Session1.vcf | wc -l
  603  grep chr1* test_Session1.vcf
  604  grep chr1* test_Session1.vcf | wc -l
  605  cutadapt --help
  606  sudo apt install cutadapt
  607  cutadapt --help
  608  head test_Session1.vcf
  609  head -1 test_Session1.vcf
  610  grep -c CTAGAG test_Session1.vcf
  611  cd /mnt/c/Users/nada_ahmed/Downloads/
  612  cd /mnt/c/Users/nada_ahmed/
  613  cd /mnt/c/Users/
  614  ls
  615  ls ./nahme/
  616  cd /mnt/c/Users/nahme/Downloads/
  617  ls
  618  cd 
  619  cd module5/
  620  ls
  621  grep chr1 test_Session1.vcf | wc -l
  622  grep chr12 test_Session1.vcf | wc -l
  623  grep 'chr12' test_Session1.vcf | wc -l
  624  grep 'chr1' test_Session1.vcf | wc -l
  625  grep 'chr13' test_Session1.vcf | wc -l
  626  grep 'chr14' test_Session1.vcf | wc -l
  627  grep 'chr15' test_Session1.vcf | wc -l
  628  grep 'chr16' test_Session1.vcf | wc -l
  629  grep 'chr111' test_Session1.vcf | wc -l
  630  grep 'chr20' test_Session1.vcf | wc -l
  631  grep \bchr1\b test_Session1.vcf | wc -l
  632  grep chr1 test_Session1.vcf | wc -l
  633  cut -d'' -f4,5 test_Session1.vcf
  634  cut -d'\t' -f4,5 test_Session1.vcf
  635  cut -d\t -f4,5 test_Session1.vcf
  636  cut -d'' -f4,5 test_Session1.vcf
  637  cut --help
  638  more test_Session1.vcf
  639  grep miRNA test_Session1.vcf
  640  grep miRNA test_Session1.vcf | wc -l
  641  grep CTAGAG test_Session1.vcf | wc -l
  642  grep CTAGAG test_Session1.vcf
  643  cut -f5 test_Session1.vcf
  644  cut -f5 test_Session1.vcf > test_ALT.vcf
  645  ls
  646  more test_ALT.vcf 
  647  cut -f4 test_Session1.vcf > test_REF.vcf
  648  ls
  649  more test_REF.vcf 
  650  more test.vcf 
  651  more test_Session1.vcf
  652  grep unprocessed_ pseudogene test_Session1.vcf
  653  cut -d';' -f1,2 test_Session1.vcf
  654  cut -d';' -f2 test_Session1.vcf
  655  cut -f8 test_Session1.vcf
  656  cut -f7 test_Session1.vcf
  657  cut -f6 test_Session1.vcf
  658  cut -f3 test_Session1.vcf
  659  cut -f2 test_Session1.vcf
  660  less test_Session1.vcf
  661  cut -f8 test_Session1.vcf
  662  cut -f9 test_Session1.vcf
  663  more test_Session1.vcf
  664  cut -f10 test_Session1.vcf
  665  cut -f10 test_Session1.vcf | cut -d':' -1 test_Session1.vcf
  666  cut -f10 test_Session1.vcf | cut -d':' -f1 test_Session1.vcf
  667  more test_Session1.vcf
  668  cut -f8 test_Session1.vcf
  669  cut -f9 test_Session1.vcf
  670  cut -f9 test_Session1.vcf | cut -d':' -f1 test_Session1.vcf
  671  cut -f9 test_Session1.vcf | cut -d':' -f1 test_Session1.vcf > GT.vcf
  672  cat GT.vcf 
  673  grep -v ## test_Session1.vcf
  674  grep -v '##' test_Session1.vcf
  675  grep -v '##' test_Session1.vcf > session1_noheader.vcf
  676  more session1_noheader.vcf 
  677  ls
  678  rm GT.vcf test_ALT.vcf test_REF.vcr session1_noheader.vcf 
  679  rm GT.vcf test_ALT.vcf test_REF.vcf session1_noheader.vcf 
  680  rm test_REF.vcf
  681  ls
  682  more test_Session1.vcf
  683  cut -f9 test_Session1.vcf
  684  cut -d':' -f1 test_Session1.vcf
  685  more test_Session1.vcf
  686  cut -f10 test_Session1.vcf | cut -d':' -f1 test_Session1.vcf
  687  cut -f10 test_Session1.vcf | cut -d':' -f1 test_Session1.vcf > GT_sample1.vcf
  688  more GT_sample1.vcf 
  689  cut -f5 test_Session1.vcf
  690  cut -d';' -f2 test_Session1.vcf
  691  cut -f10 test_Session1.vcf
  692  cut -f10 test_Session1.vcf > sample1.vcf
  693  more sample1.vcf 
  694  cut -d':' -f1 sample1.vcf 
  695  cut -d':' -f1 sample1.vcf > gt_sample1.vcf
  696  more gt_sample1.vcf 
  697  ls
  698  rm GT_sample1.vcf 
  699  ls
  700  cut -f8 test_Session1.vcf
  701  cut -d'|' -f3 test_Session1.vcf
  702  cut -d'|' -f1 test_Session1.vcf
  703  cut -f8 test_Session1.vcf > info.vcf
  704  more info.vcf 
  705  cut -d'|' -f-1 info.vcf 
  706  cut -d';' -f16 test_Session1.vcf
  707  cut -d';' -f16 info.vcf 
  708  grep unprocessed_ pseudogene test_Session1.vcf
  709  grep unprocessed_ pseudogene info.vcf 
  710  grep unprocessed_pseudogene info.vcf 
  711  grep unprocessed_pseudogene test_Session1.vcf > filtered.vcf
  712  more filtered.vcf 
  713  cut -f8 filtered.vcf 
  714  cut -d';' -f16 filtered.vcf 
  715  cut -d';' -f16 filtered.vcf > eff.vcf
  716  cut -d'|' -f5 eff.vcf 
  717  grep  eff.vcf 
  718  cut -d'|' -f8 eff.vcf 
  719  ls
  720  rm eff.vcf filtered.vcf gt_sample1.vcf info.vcf sample1.vcf 
  721  ls
  722  more test_Session1.vcf
  723  ls
  724  head -n20 test_Session1.vcf
  725  more test_Session1.vcf
  726  grep CTAGAG test_Session1.vcf
  727  grep CTAGAG test_Session1.vcf | wc -l
  728  more test_Session1.vcf
  729  grep chr1 test_Session1.vcf
  730  grep chr1 test_Session1.vcf | wc -l
  731  grep unprocessed_ pseudogene test_Session1.vcf
  732  grep unprocessed_pseudogene test_Session1.vcf
  733  more test_Session1.vcf
  734  cut -f10 test_Session1.vcf
  735  cut -f10 test_Session1.vcf | cut -d':' -f1 > test_Session1_GTsample1.vcf
  736  ls
  737  cat test_Session1_GTsample1.vcf 
  738  ls
  739  rm test_Session1_GTsample1.vcf 
  740  cut -f10 test_Session1.vcf | cut -d':' -f1 > test_Session1_GTsample1.vcf
  741  cat test_Session1_GTsample1.vcf 
  742  ls c:/
  743  ls c/:
  744  ls C:
  745  cd C:\Users\nahme\Downloads\test_Session1.vcf
  746  cp /mnt/c/users/nahme/downloads/test_Session1.vcf .
  747  ls
  748  rm test_Session1.vcf test_Session1_GTsample1.vcf 
  749  ls
  750  rm test_Session1.vcf\:Zone.Identifier 
  751  ls
  752  cp /mnt/c/users/nahme/t
  753  cp /mnt/c/users/nahme/downloads/test_Session1.vcf .
  754  ls
  755  cat test_Session1.vcf 
  756  head -n 20 test_Session1.vcf 
  757  grep CTAGAG test_Session1.vcf 
  758  grep CTAGAG test_Session1.vcf | wc -l
  759  grep chr1 test_Session1.vcf | wc -l
  760  grep chr1 test_Session1.vcf
  761  grep unprocessed_ pseudogene test_Session1.vcf 
  762  grep unprocessed_ pseudogene test_Session1.vcf | wc -l
  763  grep unprocessed_pseudogene test_Session1.vcf | wc -l
  764  more test_Session1.vcf 
  765  cut -f9 test_Session1.vcf 
  766  more test_Session1.vcf 
  767  cut -f10 test_Session1.vcf | cut -d':' -f1
  768  more test_Session1.vcf 
  769  rm test_Session1.vcf 
  770  ls
  771  cp /mnt/c/users/nahme/downloads/test_Session1.vcf .
  772  ls
  773  more test_Session1.vcf 
  774  head -n 20 test_Session1.vcf 
  775  grep CTAGAG test_Session1.vcf 
  776  grep CTAGAG test_Session1.vcf | wc -l
  777  grep chr1 test_Session1.vcf | wc -l
  778  grep chr1 test_Session1.vcf
  779  grep unprocessed_pseudogene test_Session1.vcf 
  780  more test_Session1.vcf 
  781  cut -f9 test_Session1.vcf 
  782  more test_Session1.vcf 
  783  cut -f10 test_Session1.vcf | cut -d':' -f1
  784  ls .
  785  rm test_Session1.vcf 
  786  ls
  787  cp /mnt/c/users/nahme/downloads/test_Session1.vcf .
  788  ls
  789  more test_Session1.vcf 
  790  head -n 20 test_Session1.vcf 
  791  grep CTAGAG test_Session1.vcf 
  792  grep CTAGAG test_Session1.vcf | wc -l
  793  grep chr1 test_Session1.vcf | wc -l
  794  grep chr1 test_Session1.vcf
  795  grep unprocessed_pseudogene test_Session1.vcf 
  796  more test_Session1.vcf 
  797  cut -f9 test_Session1.vcf 
  798  more test_Session1.vcf 
  799  cut -f10 test_Session1.vcf | cut -d':' -f1
  800  cut -f10 test_Session1.vcf | cut -d':' -f1 > test_Session1_GTsample1.vcf
  801  ls
  802  cat test_Session1_GTsample1.vcf 
  803  cp /mnt/c/users/nahme/downloads/genomics.human_genetic_variation.TaskVEP.vcf .
  804  pwd
  805  more genomics.human_genetic_variation.TaskVEP.vcf 
  806  pwd
  807  mkdir IBT_consolidation_week_2017
  808  ls
  809  rm genomics.human_genetic_variation.TaskVEP.vcf 
  810  mv ./IBT_consolidation_week_2017/ ./module5/
  811  cd ./module5/
  812  ls
  813  cd ./IBT_consolidation_week_2017/
  814  mv . ~
  815  mv ../IBT_consolidation_week_2017/ ~
  816  cd 
  817  ls
  818  cd IBT_consolidation_week_2017/
  819  cp mnt/c/users/nahme/downloads/Venter_chr123 _1_ .
  820  cp mnt/c/users/nahme/downloads/Venter_chr123_1_ .
  821  cp /c/users/nahme/downloads/Venter_chr123_1_ .
  822  cp /mnt/c/users/nahme/downloads/Venter_chr123_1_ .
  823  ls /mnt/c
  824  ls /mnt/c/users
  825  ls /mnt/c/users/nahme
  826  ls /mnt/c/users/nahme/downloads
  827  cp /mnt/c/users/nahme/downloads/Venter_chr123_1_.vcf .
  828  ls
  829  more Venter_chr123_1_.vcf 
  830  grep chr1 Venter_chr123_1_.vcf 
  831  grep chr1 Venter_chr123_1_.vcf > Venter_chr1.vcf
  832  ls
  833  head Venter_chr1.vcf
  834  tail Venter_chr1.vcf 
  835  tail Venter_chr123_1_.vcf 
  836  grep --help
  837  ls -l
  838  cp /mnt/c/users/nahme/downloads/Venter_chr1_VEP.vcf
  839  cp /mnt/c/users/nahme/downloads/Venter_chr1_VEP.vcf .
  840  ls
  841  ls -l
  842  head Venter_chr1_VEP.vcf 
  843  grep ENSG00000230021 Venter_chr1_VEP.vcf | wc -l
  844  grep ENSG00000230021 Venter_chr1_VEP.vcf
  845  grep CNV Venter_chr1_VEP.vcf 
  846  cd ./IBT_consolidation_week_2017/
  847  ls
  848  rm Venter_chr1.vcf Venter_chr123_1_.vcf 
  849  ls
  850  cp /mnt/c/users/nahme/downloads/Venter_chr123_1_.vcf .
  851  cp /mnt/c/users/nahme/downloads/Venter_chr123_1_ .
  852  cp /mnt/c/users/nahme/downloads/Venter_chr123_1_.vcf.vcf .
  853  cp /mnt/c/users/nahme/downloads/Venter_chr123_1_.vcf .
  854  ls
  855  cd ./IBT_consolidation_week_2017/
  856  ls
  857  head Venter_chr123_1_.vcf 
  858  tail Venter_chr123_1_.vcf 
  859  ls
  860  grep ENSG00000230021 Venter_chr1_VEP.vcf 
  861  grep ENSG00000230021 Venter_chr1_VEP.vcf | wc -l
  862  grep -c ENSG00000230021 Venter_chr1_VEP.vcf 
  863  wc -l Venter_chr1_VEP.vcf 
  864  grep ENSG00000230021 Venter_chr1_VEP.vcf 
  865  tail -n 1 Venter_chr1_VEP.vcf 
  866  grep ENSG00000230021 Venter_chr1_VEP.vcf > gene1
  867  tail -n 1 gene1 
  868  mkdir nada_EA
  869  ls
  870  cd ./biocomputing
  871  mkdir biocomputing && cd biocomputing
  872  ls -r
  873  ls -R
  874  ls .
  875  ls ~
  876  wget https://raw.githubusercontent.com/josoga2/dataset-repos/main/wildtype.fna
  877  wget https://raw.githubusercontent.com/josoga2/dataset-repos/main/wildtype.g
  878  wget https://raw.githubusercontent.com/josoga2/dataset-repos/main/wildtype.gbk
  879  ls
  880  mv wildtype.fna ../nada_EA/
  881  ls ../nada_EA/
  882  rm wildtype.gbk1
  883  ls
  884  rm wildtype.gbk.1
  885  ls
  886  #To check for mutant ("TATATATA")
  887  grep "TATATATA" wildtype.gbk 
  888  grep "TATA" wildtype.gbk wildtype.gbk 
  889  ; Q
  890  grep "tatatata" wildtype.gbk 
  891  grep "tata" wildtype.gbk 
  892  grep "tatatata" wildtype.gbk > mutant_lines
  893  ls
  894  head mutant_lines 
  895  wget https://www.ncbi.nlm.nih.gov/nuccore/NC_000913.3?report=fasta&from=363231&to=366305&strand=true
  896  ls
  897  ls -lh
  898  nano NC_000913.3\?report\=fasta 
  899  rm NC_000913.3\?report\=fasta 
  900  ls
  901  rm wget-log 
  902  ls
  903  wget https://www.ncbi.nlm.nih.gov/sviewer/viewer.cgi?db=nuccore&id=NC_000913.3&report=fasta
  904  $ wget -q -O - "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore&id=NC_000913.3&rettype=fasta"
  905  nano wildtype.gbk 
  906  tail wildtype.gbk 
  907  sudo apt-get install entrez-direct
  908  passwd
  909  cmd.exe
  910  sudo echo hi
  911  passwd username
  912  cmd.exe
  913  sudo echo hi
  914  cd ~
  915  ls
  916  rm IBT
  917  rm Coursera/
  918  rm -r Coursera/
  919  rm -r IBT2016/
  920  ls
  921  rm -r MasterCourses/
  922  rm -r module5/
  923  ls
  924  rm -r IBT_consolidation_week_2017/
  925  rm -r biocomputing/ nada_EA/
  926  ls
  927  echo lacZ - galactosidase
  928  sudo apt-get install entrez-direct
  929  cmd.exe
  930  sudo echo hi
  931  sudo apt-get install entrez-direct
  932  conda activate base
  933  conda config --add channels r
  934  ls
  935  cd miniconda3/
  936  conda activate base
  937  ls
  938  ~/miniconda3/bin/conda init
  939  Q
  940  q
  941  source ~/.bashrc
  942  conda activate base
  943  conda --version
  944  conda create --name funtools
  945  conda activate funtools
  946  conda install -c conda-forge figlet
  947  sudo apt-get install figlet
  948  figlet Hello World
  949  figlet AMR <3 Fighters
  950  figlet AMR-<3-Fighters
  951  figlet "AMR <3 Fighters"
  952  figlet "AMR ... Fighters"
  953  conda install -c bioconda bwa
  954  conda install -c bioconda blast
  955  conda install -c bioconda samtools
  956  conda install -c bioconda bedtools
  957  conda install spades
  958  conda install -c bioconda spades
  959  conda install -c bioconda bcftools
  960  conda install -c bioconda fastp
  961  conda install -c bioconda multiqc
  962  conda create -n multiqc-env python=3.9
  963  conda activate multiqc-env
  964  conda install -c bioconda multiqc
  965  conda create -n multiqc-env python=3.5
  966  cond activate multiqc-env
  967  conda activate multiqc-env
  968  conda install -c bioconda multiqc
  969  nano codes_history
  970  nano codes_history.sh
  971  history > codes.sh
